articles/141_static_remapping.md

```diff
@@ -340,7 +339,7 @@ Within each category, the rules are applied in the order in which the user gave
 
 - A node has name `talker`
 - A user gives the rules `talker:__ns:=/my_namespace` then `talker:__node:=foo`
-- The final namespace is the default `/` because the node name remap is applied before the namespace remap
+- The final node name is `foo` because the namespace and node name remap are applied without affecting each other
```

Comment:
What's the final fully qualified name? Current design says it would be `/foo` because the node name remapping happened first. Is this PR proposing the final output would be `/my_namespace/foo`?

Reply:
> What's the final fully qualified name?

It's `/my_namespace/foo` which you had already answered. I should make the document clear, I'll update it.